core/dragging/block_drag_strategy.ts

```diff
@@ -38,6 +38,8 @@ interface ConnectionCandidate {
 export class BlockDragStrategy implements IDragStrategy {
   private workspace: WorkspaceSvg;
 
+  private inGroup: boolean = false;
```

Comment:
I think this deserves a JSDoc to explain that it's for, since the name + type is not self-explanatory:

```suggestion
  /** Was there already an event group in progress when the drag started? */
  private inGroup: boolean = false;
```
(Here and in the other similar cases below.)

Also, for consistency with the other classes, I think it might be better to move this declaration to just after the declaration of `private dragging`, below.